modules/nf-core/vcfanno/main.nf

```diff
@@ -31,7 +34,8 @@ process VCFANNO {
         ${lua_cmd} \\
         ${toml} \\
         ${vcf} \\
-        > ${prefix}.vcf
+        | bgzip ${args2} --threads ${task.cpus} > ${prefix}.vcf.gz \\
+        && tabix ${args3} ${prefix}.vcf.gz
```

Comment:
What's the reason for the &&? Cna't it be a standalone?

Reply:
It's simply to make the stderr less verbose in case of an error :). We don't need tabix to tell us the vcf is incorrect if something went wrong earlier